internal/services/formatters/go/gosec/formatter_test.go

```diff
@@ -24,112 +24,184 @@ import (
 
 	"github.com/ZupIT/horusec-devkit/pkg/entities/analysis"
 	"github.com/ZupIT/horusec-devkit/pkg/enums/tools"
-	cliConfig "github.com/ZupIT/horusec/config"
-	"github.com/ZupIT/horusec/internal/entities/workdir"
+	"github.com/ZupIT/horusec/config"
 	"github.com/ZupIT/horusec/internal/services/docker"
 	"github.com/ZupIT/horusec/internal/services/formatters"
 )
 
-func TestGoLang_StartAnalysis(t *testing.T) {
-	t.Run("Should run analysis and return error because output is empty", func(t *testing.T) {
-		dockerAPIControllerMock := &docker.Mock{}
-		dockerAPIControllerMock.On("SetAnalysisID")
-		dockerAPIControllerMock.On("CreateLanguageAnalysisContainer").Return("", nil)
+func TestGosecStartAnalysis(t *testing.T) {
+	t.Run("Should run analysis with sucessfull when output is empty", func(t *testing.T) {
```

Comment:
I guess that `Should run analysis successfully when output is empty` is more accurate.
If decided to keep the message, there's a typo in `sucessfull` -> `successful`.

Reply:
Done.